dll/cpl/desk/settings.c

```diff
@@ -474,6 +474,7 @@ OnBPPChanged(IN HWND hwndDlg, IN PSETTINGS_DATA pData)
     hSpectrumDC = GetDC(hSpectrumControl);
```

Comment:
Check `hSpectrumDC != NULL` here too?

Reply:
Yes

Reply:
This is _not_ what I suggested. Explain the change of behavior!